libraries/exporters/src/main/java/org/eclipse/sw360/exporter/helper/ReleaseHelper.java

```diff
@@ -211,15 +212,30 @@ private void addFieldValueToRow(List<String> row, Release._Fields field, Release
                 addReleaseIdToRelationShipToRow(release.getReleaseIdToRelationship(), row);
                 break;
             case ATTACHMENTS:
-                String size = Integer.toString(release.isSetAttachments() ? release.getAttachments().size() : 0);
-                row.add(size);
+                if(isForProjectExport) {
+                    addSourceAttachmentsInformationToRow(release, row);
+                    }else {
+                    String size = Integer.toString(release.isSetAttachments() ? release.getAttachments().size() : 0);
+                    row.add(size);
+                }
                 break;
             default:
                 Object fieldValue = release.getFieldValue(field);
                 row.add(fieldValueAsString(fieldValue));
         }
-    }
+    }    
+    
+    private void addSourceAttachmentsInformationToRow(Release release, List<String> row) throws SW360Exception{
+        Set<Attachment> attachments = release.getAttachments() != null ? release.getAttachments() : Collections.emptySet();
 
+        String result = attachments.stream()
+                .filter(attachment -> attachment.getAttachmentType().name().equals("SOURCE") || attachment.getAttachmentType().name().equals("SOURCE_SELF"))
```

Comment:
I suggest to change the filter as below.
Define a Predicate:
```java
final Predicate<Attachment> isSourceAttachment = attachment -> AttachmentType.SOURCE.equals(attachment.getAttachmentType()) || AttachmentType.SOURCE_SELF.equals(attachment.getAttachmentType());
```
Use the predefined Predicate in the filter.
```java
.filter(isSourceAttachment)
```